react/react1/week1/vite-project/src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
+import { TodoList, todo } from './toDoList'
+
+function App() {
+  const [count, setCount] = useState(0)
+
+  return (
+    <div className="App">
+      <div>
+        <a href="https://vitejs.dev" target="_blank">
+          <img src={viteLogo} className="logo" alt="Vite logo" />
+        </a>
+        <a href="https://reactjs.org" target="_blank">
+          <img src={reactLogo} className="logo react" alt="React logo" />
+        </a>
+      </div>
+      <h1>Homework</h1>
+      <div className="card">
+        <button onClick={() => setCount((count) => count + 1)}>
+          count is {count}
+        </button>
+        <h2>To do list</h2>
+      </div>
+      <TodoList todo={todo}></TodoList>
```

Comment:
you can add a closing tag to your component, making it more readable. So from `<TodoList todo={todo}></TodoList>` to `<TodoList todo={todo}/>`